pkg/clusteragent/clusterchecks/stats.go

```diff
@@ -56,12 +56,16 @@ func (h *Handler) getStats() *types.Stats {
 func (d *dispatcher) getStats() *types.Stats {
 	d.store.RLock()
 	defer d.store.RUnlock()
-
+	var checkNames []string
+	for _, m := range d.store.digestToConfig {
+		checkNames = append(checkNames, m.Name)
+	}
```

Comment:
I think it should be a set, for faster lookup later and because it's expected to have multiple instances of the same check, so a set will avoid reporting duplicated check names.

Reply:
Makes sense 👍

Reply:
added in: cf5bb9b3dbc7706bf6b151a898af1abccd3d46c9